portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,28 @@
 #greeting-container {
   margin-top: 20px;
 }
+body{
```

Comment:
"Use a space between the last selector and the declaration block." - https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation

Here and elsewhere there's a `{`.